app/models/match_exporter.rb

```diff
@@ -7,7 +7,7 @@ def initialize(account:, season:)
   def export
     column_names = ['Rank', 'Map', 'Comment', 'Day', 'Time', 'Heroes', 'Ally Leaver',
                     'Ally Thrower', 'Enemy Leaver', 'Enemy Thrower', 'Group',
-                    'Placement', 'Result']
+                    'Placement', 'Result', 'POTG']
```

Comment:
How about making “ot” lowercase since they represent a preposition and an article?